integration/e2e/composite_service.go

```diff
@@ -61,11 +61,11 @@ func (s *CompositeHTTPService) WaitSumMetrics(isExpected func(sums ...float64) b
 }
 
 // SumMetrics returns the sum of the values of each given metric names.
-func (s *CompositeHTTPService) SumMetrics(metricNames ...string) ([]float64, error) {
+func (s *CompositeHTTPService) SumMetrics(metricNames []string, opts ...MetricsOption) ([]float64, error) {
```

Comment:
I get the use case, plus, sometimes you really want only certain label, so we could have `WithLabelEquals(name, value string)` 

However what do you think about simplifying interfaces and having just single metric name? We can save on `[]string{` boilerplate :thinking:

Reply:
> However what do you think about simplifying interfaces and having just single metric name? We can save on []string{ boilerplate

We could, but then the testers `*AmongTwo` wouldn't work anymore. Cortex is not using it, but Thanos does. Thoughts?

Reply:
> so we could have WithLabelEquals(name, value string)

Agree, done.